connect/connect-c7/README.md

```diff
@@ -4,3 +4,48 @@
 
 ### Worker
 
+#### Adjusting Retry Count for Camunda 7 Workers
+
+This feature allows you to adjust the retry count for Camunda 7 workers. With the introduction of the `getRemainingRetries` method in `Camunda7WorkerAdapter`, developers can control the number of retries applied to an external task.
+
+##### Usage
+
+The `getRemainingRetries` method is used to determine the remaining number of retries for a task. It considers various sources to determine the retry count.
+
+##### Priority Order
+
+The `getRemainingRetries` method follows a specific priority order when determining the retry count:
+
+1. **BPMN Retries** (Process Input Variables): If the BPMN definition of the task has specified retries and this information is available as input variables, the BPMN retries will be used.
+
+2. **Overridden Default Value**: If no BPMN retries are present, an overridden default value can be used. This value can be set using an input variable (process variable).
+
+3. **Default Value**: If neither BPMN retries nor an overridden default value is present, a default value of 3 will be used.
+
+##### Changing the Retry Count
+
+To adjust the retry count, you can define the corresponding input variable (`retries`) for the worker. This input variable can be set in the BPMN diagram and will be recognized by the `Camunda7WorkerAdapter`.
```

Comment:
@rami-nk, this is something for us to add to the documentation as well. 💭 
We just need to keep that in our mind. 👍

Reply:
Yess this is a good point. I'm not sure where it fits better, because in our documentation we do not have separate sections for c7 and c8. Would you say that this is worth thinking about?

Reply:
Hm, I would add this as a note to the existing documentation. 🤔 
Let's chat about this over in the docs repo!